core-library/src/main/java/org/silverpeas/core/admin/persistence/ComponentInstanceTable.java

```diff
@@ -236,16 +237,18 @@ protected void prepareInsert(String insertQuery, PreparedStatement insert,
     insert.setInt(17, row.inheritanceBlocked);
   }
 
-  public void updateComponentOrder(int componentId, int orderNum) throws AdminPersistenceException {
+  public void updateComponentOrder(int componentId, int orderNum) throws SQLException {
     int[] values = new int[]{orderNum, componentId};
     updateRelation(UPDATE_COMPONENT_ORDER, values);
   }
 
-  static final private String UPDATE_COMPONENT_ORDER =
-      "update ST_ComponentInstance set" + " orderNum = ?" + " where id = ?";
+  public static final String UPDATE_ST_COMPONENT_INSTANCE_SET = "update ST_ComponentInstance set";
+  public static final String WHERE_ID = " where id = ?";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1213)